packages/discord.js/typings/index.d.ts

```diff
@@ -2453,6 +2453,7 @@ export class Webhook extends WebhookMixin() {
   public sourceChannel: NewsChannel | APIPartialChannel | null;
   public token: string | null;
   public type: WebhookType;
+  public application_id?: Snowflake;
```

Comment:
Shouldn't it be this?

```suggestion
  public applicationId: Snowflake | null;
```